composer.json

```diff
@@ -19,6 +19,10 @@
         "symfony/css-selector":  "~2.0"
     },
 
+    "require-dev": {
+        "symfony/phpunit-bridge": "~2.7"
```

Comment:
I've read http://symfony.com/blog/new-in-symfony-2-7-phpunit-bridge and now I get the clear picture.